enterprise/frontend/src/metabase-enterprise/moderation/constants.ts

```diff
@@ -0,0 +1,26 @@
+import type { ColorName } from "metabase/lib/colors/types";
+import type { IconName } from "metabase/ui";
+
+export const MODERATION_STATUS = {
+  verified: "verified",
+};
+
+export const MODERATION_STATUS_ICONS: Map<
+  string | null,
+  { name: IconName; color: ColorName }
+> = new Map();
```

Comment:
why use a map here instead of a plain object?

Reply:
It needs to map `null`. That was how the system was designed before and I didn't want to change it too much